### PR TITLE
feat(reminder-scheduler): adds reminder-scheduler component

### PR DESCRIPTION
**Pull request by prince981620**
hey @hari apologies for my previous PR i mistakenly created branch from my old branch.
please review.
thank-you

### CODE REVIEW

File: animata/card/reminder-scheduler.tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    if (check) {
+      setTimeout(() => {
+        setActive((current) => !current);
+      }, 100);
+    } else {
+      setTimeout(() => {
+        setActive((current) => !current);
+      }, 100);
+    }
+  }, [check]);
```
Comment (hari) [inline animata/card/reminder-scheduler.tsx:106]: you need to clear timeout on unmount.

File: content/docs/card/reminder-scheduler.mdx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<ComponentPreview name="card-reminder-scheduler--docs" />
+
+## Installation
+
+<Steps>
+<Step>Install dependencies</Step>
+
+```bash
+npm install
+```
```
Comment (hari) [inline content/docs/card/reminder-scheduler.mdx:17]: remove the step if it is not necessary

File: content/docs/card/reminder-scheduler.mdx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+     },
+       transitionTimingFunction: {
+        slow: "cubic-bezier(.405, 0, .025, 1)",
+        "minor-spring": "cubic-bezier(0.18,0.89,0.82,1.04)",
+     }
+    },
+  },
+```
+
+````
```
Comment (hari) [inline content/docs/card/reminder-scheduler.mdx:37]: extra backtick

File: animata/card/reminder-scheduler.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+const ReminderScheduler: React.FC = () => {
+  const [isRepeating, setIsRepeating] = useState<boolean>(true);
+  const [repeatInterval, setRepeatInterval] = useState<string>("Weekly");
+  const [selectedDays, setSelectedDays] = useState<string[]>(["Mo", "We", "Th"]);
+
+  const daysOfWeek = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"];
+
+  const toggleRepeating = () => {
+    console.log("Toggled");
```
Comment (hari) [inline animata/card/reminder-scheduler.tsx:13]: remove log

File: animata/card/reminder-scheduler.tsx [hunk 5]
```diff
@@ -0,0 +1,137 @@
+import React, { useEffect, useState } from "react";
+
+import { cn } from "@/lib/utils";
+
+const ReminderScheduler: React.FC = () => {
+  const [isRepeating, setIsRepeating] = useState<boolean>(true);
+  const [repeatInterval, setRepeatInterval] = useState<string>("Weekly");
+  const [selectedDays, setSelectedDays] = useState<string[]>(["Mo", "We", "Th"]);
+
+  const daysOfWeek = ["Mo", "Tu", "We", "Th", "Fr", "Sa", "Su"];
```
Comment (hari) [inline animata/card/reminder-scheduler.tsx:10]: you can move this out of the component

File: animata/card/reminder-scheduler.tsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+
+  const toggleRepeating = () => {
+    console.log("Toggled");
+    isRepeating ? setSelectedDays(["Th", "Fr", "Su"]) : setSelectedDays(["Mo", "We", "Sa"]);
```
Comment (hari) [inline animata/card/reminder-scheduler.tsx:14]: you don't need a state for this, simply use a local variable on top `const selectedDays = isRepeating ? ...`

File: animata/card/reminder-scheduler.tsx [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+        </select>
+      </div>
+
+      {/* Day Selection */}
+      <div className="flex items-center rounded-2xl bg-gray-100 p-4">
+        <div className="grid grid-cols-7 justify-around gap-2">
+          {daysOfWeek.map((day) => (
+            <SwapText
+              key={day}
+              check={selectedDays.includes(day)}
```
Comment (hari) [inline animata/card/reminder-scheduler.tsx:49]: you can use `Set` instead of `Array` for `selectedDays`.

**Review by hari — CHANGES_REQUESTED** [review]

**Comment by hari** [comment]
@prince981620 Could you merge the latest main on your branch?

File: content/docs/card/reminder-scheduler.mdx [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+title: Reminder Scheduler
+description: a simple card to set up recurring reminders along with frequency and repetition.
+author: m_jinprince
+labels: ["requires interaction", "toggle switch"]
+---
+
+<ComponentPreview name="card-reminder-scheduler--docs" />
+
+## Installation
+
```
Comment (hari) [inline content/docs/card/reminder-scheduler.mdx:11]: `<Steps>` is missing here @prince981620

**Comment by prince981620** [comment]
> @prince981620 Could you merge the latest main on your branch?

merged and pushed again please review.

**Review by hari — APPROVED** [review]

**Comment by prince981620** [comment]
hey @sudhashrestha i have already completed #296 and this also got merged: apologies for not mentioning it up there

**Pull request merged**